### PR TITLE
New MD file

### PR DESCRIPTION
**Pull request by BikoNolivar**

### CODE REVIEW

File: git/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[.ShellClassInfo]
+InfoTip=This folder is shared online.
```
Comment (myeong) [inline git/desktop.ini:2]: hey I don't need this file

File: git/introductions/bolivar.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Nikoli Bolivar's ReadME file.
+Welcome.
```
Comment (myeong) [inline git/introductions/bolivar.md:2]: I like this

**Comment by BikoNolivar** [comment]
is there a reason this "desktop.ini" was added/pushed? i followed your
instructions in class and it seems I am the only one that also added this
"desktop.ini" file

**Comment by myeong** [comment]
@BikoNolivar it is possible that your computer automatically created it. I am not sure exactly why it was created in the folder. If you see this kind of unexpected files in the folder, you can either list the file name in the ".gitignore" or just delete it before you push.

**Pull request merged**